File: src/tool/spring-boot-generator/output/test/db/sysSystemDictionary.js
let sysSystemDictionary = {
  name: "sysSystemDictionary",
  description: "",
  columns: {
    dicid: {
      name: "dicid",
      description: "瀛楀吀Id",
      isPk: true,
      autoIncrement: true,
      nullable: false,
      type: {
        name: "int",
        length: null
      }
    },
    enumName: {
      name: "enumName",
      description: "",
      nullable: true,
      type: {
        name: "varchar",
        length: null
      }
    },
    enumType: {
      name: "enumType",
      description: "鏋氫妇绫诲瀷",
      nullable: false,
      type: {
        name: "varchar",
        length: null
      }
    },
    text: {
      name: "text",
      description: "鏋氫妇椤瑰叿浣撳悕瀛?",
      nullable: false,
      type: {
        name: "varchar",
        length: null
      }
    },
    value: {
      name: "value",
      description: "鏋氫妇value",
      nullable: false,
      type: {
        name: "varchar",
        length: null
      }
    },
    sortId: {
      name: "sortId",
      description: "鎺掑簭",
      nullable: false,
      type: {
        name: "int",
        length: null
      }
    },
    status: {
      name: "status",
      description: "鐘舵€?0-鍚敤,1-绂佺敤",
      nullable: false,
      type: {
        name: "int",
        length: null
      }
    },
    systemId: {
      name: "systemId",
      description: "浣跨敤绯荤粺",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    }
  }
};

let columnsArray = [
  "dicid",
  "enumName",
  "enumType",
  "text",
  "value",
  "sortId",
  "status",
  "systemId"
];

function create() {
  return {
  dicid: 806,
  enumName: "142",
  enumType: "鏋氫妇绫诲瀷163",
  text: "鏋氫妇椤瑰叿浣撳悕瀛?303",
  value: "鏋氫妇value14",
  sortId: 553,
  status: 401,
  systemId: 947
};
}

sysSystemDictionary.columnsArray=columnsArray;
sysSystemDictionary.create=create;
module.exports=sysSystemDictionary;

File: src/tool/spring-boot-generator/output/railway/railway/src/main/java/com/railway/pojo/entity/SysSystemDictionary.java
package com.railway.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;



@ApiModel("")
@Getter
@Setter
public class SysSystemDictionary  {

    @ApiModelProperty("瀛楀吀Id")
    private Integer dicid;

    @ApiModelProperty("")
    private String enumName;

    @ApiModelProperty("鏋氫妇绫诲瀷")
    private String enumType;

    @ApiModelProperty("鎺掑簭")
    private Integer sortId;

    @ApiModelProperty("鐘舵€?0-鍚敤,1-绂佺敤")
    private Integer status;

    @ApiModelProperty("浣跨敤绯荤粺")
    private Integer systemId;

    @ApiModelProperty("鏋氫妇椤瑰叿浣撳悕瀛?")
    private String text;

    @ApiModelProperty("鏋氫妇value")
    private String value;

}